planner/core/util.go

```diff
@@ -231,3 +233,15 @@ func GetStatsInfo(i interface{}) map[string]uint64 {
 	statsInfos = CollectPlanStatsVersion(physicalPlan, statsInfos)
 	return statsInfos
 }
+
+// ExtractStringFromStringSet helps extract string info from set.StringSet
+func ExtractStringFromStringSet(set set.StringSet) string {
+	r := ""
+	if len(set) < 1 {
+		return r
+	}
+	for k := range set {
```

Comment:
Please add a sort here to make the result stable.

currently

```sql
information_schema> desc select * from `CLUSTER_LOG` where level in ('warn','error')
+----------------+----------+------+-------------------------------------------------------------------------------------------------------------------------------------------+
| id             | estRows  | task | operator info                                                                                                                             |
+----------------+----------+------+-------------------------------------------------------------------------------------------------------------------------------------------+
| MemTableScan_5 | 10000.00 | root | table:CLUSTER_LOG, Start_time:1970-01-01 08:00:00, end_time:1970-01-01 07:59:59.999, node_types:[], instances:[], log_levels:[warn,error] |
+----------------+----------+------+-------------------------------------------------------------------------------------------------------------------------------------------+
1 row in set
Time: 0.007s
information_schema> desc select * from `CLUSTER_LOG` where level in ('warn','error')
+----------------+----------+------+-------------------------------------------------------------------------------------------------------------------------------------------+
| id             | estRows  | task | operator info                                                                                                                             |
+----------------+----------+------+-------------------------------------------------------------------------------------------------------------------------------------------+
| MemTableScan_5 | 10000.00 | root | table:CLUSTER_LOG, Start_time:1970-01-01 08:00:00, end_time:1970-01-01 07:59:59.999, node_types:[], instances:[], log_levels:[error,warn] |
+----------------+----------+------+-------------------------------------------------------------------------------------------------------------------------------------------+
```

Reply:
I wonder is there any existed tool or function in tidb to sort the `set.String` or i need to build an extra `type` to implement the `sort`?

Reply:
updated.